iOS/DuckDuckGo/DuckPlayer/DuckPlayer.swift

```diff
@@ -824,16 +853,76 @@ final class DuckPlayer: NSObject, DuckPlayerControlling {
     }
 
     private func setupSubscriptions() {
+        // Subscribe to timestamp updates from the UserScript (YouTube.com)
+        currentTimeStampPublisher
+            .sink { [weak self] timestamp in
```

Comment:
You can do this a bit more succinctly using `.assign(to: \.youtubeTimestamp, onWeaklyHeld: self)`.